cfme/containers/provider/openshift.py

```diff
@@ -54,6 +54,15 @@ def view_value_mapping(self):
         return out
 
 
+class VirtualizationEndpoint(ServiceBasedEndpoint):
+    """Represents virtualization Endpoint"""
+    name = 'virtualization'
+
+    @property
+    def view_value_mapping(self):
+        return {'kubevirt_token': self.token}
```

Comment:
Please put a comment in-line explaining how this endpoint takes its hostname + port + sec_protocol from the default endpoint and only token can be entered.

Reply:
added.